packages/lesswrong/lib/SwitchingCollection.ts

```diff
@@ -120,6 +120,22 @@ class SwitchingCollection<T extends DbObject> {
     });
   }
 
+  addDefaultView(view: Function) {
+    this.mongoCollection.defaultView = view;
+    this.pgCollection.defaultView = view;
+  }
+
+  addView(viewName: string, view: Function) {
+    if (!this.mongoCollection.views) {
+      this.mongoCollection.views = {};
+    }
+    if (!this.pgCollection.views) {
+      this.pgCollection.views = {};
+    }
+    this.mongoCollection.views[viewName] = view;
+    this.pgCollection.views[viewName] = view;
+  }
```

Comment:
It seems that way. I was changing everything to `pg` instead of `switching` very quickly after migrating the data though, so if anything was broken (which I don't think is likely because I didn't see any errors) it would've only been for a very short amount of time.